_posts/2019-10-28-Javascript-We-Need-To-Talk.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: Javascript, We Need To Talk
+date: 2019-10-28 18:28:05.000000000 +01:00
+type: post
+published: true
+status: publish
+categories: [Programming]
+image:
+image2:
+author: Valentino Urbano
+---
+
+Javascript is everywhere and its popularity is only increasing with time. Soon enough everything will be able to run javascript, if we're not there already. Thanks to that advantage and the fact that you must[^1] write javascript to target the web most software is javascript and if you're a programmer it is a given that you will write some javascript sooner or later in your career.
+
+Being used to really strong typing and compiler checks with Swift. I don't like writing Javascript. I try to avoid it as much as possible. I feel like it is really easy to write terrible and unsafe code with it. That said I still use it pretty often and slowly it is getting more bearable (mostly thanks to [Typescript][2]).
```

Comment:
You go from runons to sentences so short they lose their meaning...

Reply:
Better?